ansible.html.markdown

```diff
@@ -452,7 +452,7 @@ Junja is powerfull. It has built-in many usefull functions.
 [Read More](http://docs.ansible.com/ansible/latest/playbooks_filters.html)
 
 ### ansible-vault
-To maintain **ifrastructure as a code** you need to store secrets. 
+To maintain **infrastructure as code** you need to store secrets. 
   Ansible provides a way to encrypt the confidential files so you can store it in the repository, yet the files are decrypted in-fly during ansible execution.
```

Comment:
Ansible provides a way to encrypt confidential files so you can store them in the repository, yet the files are decrypted on-the-fly during ansible execution.